docker-build.sh

```diff
@@ -334,7 +338,7 @@ login_to_registry() {
 }
 
 create_repos() {
-  if ! _can_push; then
+  if ! _can_push || ! _skip_create_repos; then
```

Comment:
I think the condition should actually be:

      ! _can_push || _skip_create_repos